package.json

```diff
@@ -50,15 +42,23 @@
     "less-loader": "4.0.5",
     "prettier": "1.10.2",
     "proxy-middleware": "0.15.0",
-    "story-tools": "./deps/story-tools",
     "style-loader": "0.20.1",
     "webpack": "3.10.0",
     "webpack-dev-server": "2.11.1"
   },
   "dependencies": {
+    "angular": "1.6.9",
+    "angular-bootstrap": "0.12.2",
+    "angular-bootstrap-colorpicker": "3.0.32",
+    "angular-mocks": "1.6.9",
+    "angular-sortable-view": "0.0.17",
+    "angular-translate": "2.17.0",
     "angular-animate": "1.6.9",
+    "bootstrap": "3.3.7",
+    "jquery": "3.3.1",
     "papaparse": "^4.3.7",
-    "time-controls": "git+https://git@github.com/mapstory/story-tools.git"
+    "story-tools":
+      "git+https://git@github.com/mapstory/story-tools.git#composer"
```

Comment:
the story-tools entry now needs to look like this:
`"story-tools": "./deps/story-tools"`

Reply:
Oh, and I thought `time-controls` was still needed